src/main/java/com/mig82/folders/properties/FolderPropertiesUtil.java

```diff
@@ -0,0 +1,97 @@
+package com.mig82.folders.properties;
+
+import com.cloudbees.hudson.plugins.folder.AbstractFolder;
+import hudson.model.Item;
+import hudson.model.ItemGroup;
+import hudson.model.Job;
+import jenkins.model.Jenkins;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class FolderPropertiesUtil {
```

Comment:
I'm not sure why you broke out this code into a separate class. But if you feel strongly about it, I'm not against it. Just please add the Javadoc explaining what the class does.
Also, this is not really a `FooUtil` kind of class. It only does one thing, so I suggest we rename this to `FolderPropertiesLoader`, and rename the `loadFolderProperties` function to `loadAncestorProperties`. Thoughts?